app/views/conversations/index.html.erb

```diff
@@ -0,0 +1 @@
+<%= render "nav_conversations" %>
```

Comment:
This appears to be the only place we're using the partial, can we delete that file and just put the contents of it here in index?